src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -33,71 +42,106 @@
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  * @author Saša Stamenković <umpirsky@gmail.com>
  */
-class ResourceController extends FOSRestController
+class ResourceController extends ContainerAware
```

Comment:
Why not extending symfony's controller class? We loose quite a few handy methods.